packages/trader/src/Modules/Trading/Containers/trade.tsx

```diff
@@ -205,29 +233,36 @@ const Trade = observer(() => {
                                 'vanilla-trade-chart': is_vanilla,
                             })}
                         >
-                            <ChartLoader is_visible={is_chart_loading || should_show_active_symbols_loading} />
+                            <ChartLoader
+                                is_visible={
+                                    is_chart_loading || should_show_active_symbols_loading || shouldShowPortraitLoader
+                                }
+                            />
                             <TradeChart topWidgets={topWidgets} is_accumulator={is_accumulator} />
                         </div>
                     )}
                 </React.Suspense>
             </Div100vhContainer>
-            <div className={form_wrapper_class}>
-                {is_market_closed && !is_market_unavailable_visible && (
-                    <MarketIsClosedOverlay
-                        is_eu={is_eu}
-                        is_synthetics_trading_market_available={is_synthetics_trading_market_available}
-                        onClick={onTryOtherMarkets}
-                        onMarketOpen={prepareTradeStore}
-                        symbol={symbol}
+            {!shouldShowPortraitLoader && (
+                <div className={form_wrapper_class}>
+                    {is_market_closed && !is_market_unavailable_visible && (
+                        <MarketIsClosedOverlay
+                            is_eu={is_eu}
+                            is_synthetics_trading_market_available={is_synthetics_trading_market_available}
+                            onClick={onTryOtherMarkets}
+                            onMarketOpen={prepareTradeStore}
+                            symbol={symbol}
+                        />
+                    )}
+                    <FormLayout
+                        is_market_closed={is_market_closed}
+                        is_trade_enabled={
+                            is_trade_enabled && form_components.length > 0 && network_status.class === 'online'
+                        }
                     />
-                )}
-                <FormLayout
-                    is_market_closed={is_market_closed}
-                    is_trade_enabled={
-                        is_trade_enabled && form_components.length > 0 && network_status.class === 'online'
-                    }
-                />
-            </div>
+                </div>
+            )}
+            {shouldShowPortraitLoader && <Loader isFullScreen />}
```

Comment:
you might wanna consider writing it like so:
```
shouldShowPortraitLoader ? <Loader isFullScreen /> : <div ...
```

Reply:
doesnt work, i tried before. the page didnt get load